frontend/src/features/contacts/contact/detail/Person.test.tsx

```diff
@@ -180,13 +180,14 @@ describe('Contact PersonView component', () => {
     });
 
     var phoneValueElements = component.getAllByTestId('phone-value');
-    expect(phoneValueElements.length).toBe(4);
+    expect(phoneValueElements.length).toBe(5);
 
     // Verify that the display is in the correct order
-    expect(phoneValueElements[0].textContent).toBe(phoneFormatter(workMobile.value));
+    expect(phoneValueElements[0].textContent).toBe(phoneFormatter(personalMobile.value));
```

Comment:
After a discussion with Kartik, apparently work mobile and work phone both are of type work, and only personal mobile is of type mobile. With this that means personal mobile takes precedence over the two work phone values. Waiting on confirmation again from him as I thought this seemed a little strange

Reply:
This is the first that I've heard of that, if he can point to a requirement for that in the original story then sure, but we can't be making up new requirements during FT.

Reply:
Dropping changes other than bug